contract/lib/dns.ts

```diff
@@ -1,17 +1,6 @@
 import { DNSRecord } from 'dns-js'
 import BufferWriter from 'dns-js/lib/bufferwriter'
-
-interface SOARecordValue {
-  primary: string
-  admin: string
-  serial: number
-  refresh: number
-  retry: number
-  expiration: number
-  minimum: number
-}
-// TODO: define members of object for each type of record
-type EncodedRecord = [string, Buffer, object]
+import type { EncodedRecord, ARecord, CNAMERecord, SOARecord, TXTRecord } from './types.d.ts'
```

Comment:
drop .ts. It should also be just "import"

Reply:
Done :pray: